peercrawler.py

```diff
@@ -40,15 +41,18 @@ def get_telemetry(ctx, s):
 
 class peer_manager:
     def __init__(self, ctx,
-                 verbosity=0,
+                 verbosity=0, initial_graph: dict[Peer, peer_set] = None,
```

Comment:
You could set the default to `{}` and avoid the need for an if block.

Reply:
It's really a small detail though, and since I already made it this way it'd be too bothersome to change it now.

Reply:
Some code is passing `None` to the initial_graph parameter.